src/Folder.h

```diff
@@ -93,7 +93,6 @@
 -(NSString *)lastUpdateString;
 -(NSString *)username;
 -(NSString *)password;
--(NSDictionary *)attributes;
```

Comment:
Wouldn't it be better to put his in `Folder.h`:
`@property (copy) NSDictionary *attributes` (or `NSMutableDictionary`)?